ui/app/components/ui/dropdown/dropdown.scss

```diff
@@ -0,0 +1,10 @@
+.dropdown {
+  appearance: none;
+  -moz-appearance: none;
```

Comment:
Not sure why autoprefixer didn't catch this 🤔 I had to add it in testing

Reply:
It looks like the autoprefixer does handle this rule, but it's not running in Storybook right now. I've added a note about this, and will return to it later.